workflow/scripts/create_somalier_mqc_config.py

```
@@ -14,16 +14,102 @@
+        samples_file = snakemake.input.samples
+        ped_file = snakemake.input.ped
+        mqc_config_file = snakemake.config.get("somalier_trio_mqc", {}).get("mqc_config", "")
+        rel_check_mqc_out = snakemake.output.rel_check_mqc
+        sex_check_mqc_out = snakemake.output.sex_check_mqc
+        general_stats_mqc_out = snakemake.output.general_stats_mqc
+
+        # Load somalier MultiQC config
+        somalier_mqc_configs = {}
+        if mqc_config_file:
+            with open(mqc_config_file, 'r') as report_configs:
+                somalier_mqc_configs = yaml.load(report_configs, Loader=yaml.FullLoader)
```

Ruff (0.15.14)

[error] Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)